README.md

```diff
@@ -67,5 +67,6 @@ Name              | Description                                         | Defaul
 "slugify"         | A custom slugification function                     | [string.js' `slugify`][slugify]
 "markerPattern"   | Regex pattern of the marker to be replaced with TOC | `/^\[\[toc\]\]/im`
 "listType"        | Type of list (`ul` for unordered, `ol` for ordered) | `ul`
+"fliter"          | Function headings are passed through for TOC display| `heading => heading`
```

Comment:
But I think the name of the function is a bit confusing. How would you feel about calling it `format` instead?